aiter/ops/triton/moe_op.py

```diff
@@ -212,12 +208,17 @@ def _fused_moe_kernel_gptq_awq(
             k_mask = None
             k_other = None
 
-        a = tl.load(
-            a_ptrs,
-            mask=token_mask[:, None] & (offs_k[None, :] < K - k * BLOCK_SIZE_K),
-            other=0.0,
-        )
-        b = tl.load(b_ptrs)
+            if EVEN_K:
+                a = tl.load(a_ptrs, mask=token_mask[:, None], other=0.0)
+                b = tl.load(b_ptrs)
```

Comment:
load of b is not dependent on EVEN_K. May be we can move it outside the if-else?

Reply:
On it

Reply:
We should rather add the k mask for the else branch I think? I will do it that way

Reply:
Just realized we had block_k_diviable. I integrated with it